fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseFormatType.java

```diff
@@ -1,20 +1,10 @@
-// Copyright 2016 Yahoo Inc.
+// Copyright 2017 Yahoo Inc.
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
 package com.yahoo.bard.webservice.web;
 
-import java.util.Locale;
+public interface ResponseFormatType {
 
-/**
- * Response format type.
- */
-public enum ResponseFormatType {
-    JSON,
-    CSV,
-    DEBUG,
-    JSONAPI;
+    boolean accepts(String responseFormatValue);
```

Comment:
This `accepts` pattern is great, and should serve as a fantastic base for registering additional `ResponseFormatTypes` where the 1st one "asked" wins.

I can see it needing to be extended (how, I'm not quite sure yet) to handle the "q" part of an Accept header w/ multiple content types, but that's a problem for the future and shouldn't be too painful to solve.